.github/workflows/test.yml

```diff
@@ -78,5 +78,7 @@ jobs:
 
       - name: Upload to codecov.io
         uses: codecov/codecov-action@125fc84a9a348dbcf27191600683ec096ec9021c # v4.4.1
+        with:
+          file: reports/kover/reportDebug.xml
```

Comment:
I believe this needs to be `app/build/reports/kover/reportDebug.xml` if the path is relative to the repo/project root directory.